doc/nrf/known_issues.rst

```diff
@@ -1959,35 +1968,53 @@ tx_buffer_length set incorrectly
 Trusted Firmware-M (TF-M)
 *************************
 
-.. rst-class:: v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+.. rst-class:: v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
 
 NCSDK-11195: Build errors when enabling :kconfig:`CONFIG_BUILD_WITH_TFM` option
   Configuring the BUILD_WITH_TFM configuration in SES project configuration or using ``west -t menuconfig`` results in build errors.
 
   **Workaround:** Set ``CONFIG_BUILD_WITH_TFM=y`` in project configuration file (:file:`prj.conf`) or through west command line (``west build -- -DCONFIG_BUILD_WITH_TFM=y``).
 
-.. rst-class:: v1-9-0
+.. rst-class:: v1-9-1 v1-9-0
 
 NCSDK-13530: TF-M minimal build has increased in size
   TF-M minimal build exceeds 32 kB due to increased dependencies in the TF-M crypto partition.
 
-.. rst-class:: v1-9-0
+.. rst-class:: v1-9-1 v1-9-0
```

Comment:
```suggestion
.. rst-class:: v1-9-0
```

This is fixed.

(Although not with high confidence, but we wish to document it as fixed until proven otherwise).